src/options/quantifiers_options.toml

```diff
@@ -1318,6 +1318,14 @@ name   = "Quantifiers"
   default    = "3"
   help       = "maximum number of instantiation rounds for sygus verification calls (-1 == no limit, default is 3)"
 
+[[option]]
+  name       = "sygusVerifyTimeout"
+  category   = "regular"
+  long       = "sygus-verify-timeout=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "timeout (in milliseconds) for verifying satisfiability of synthesized terms"
```

Comment:
```suggestion
  help       = "timeout (in milliseconds) for verifying satisfiability of synthesized terms. setting the timeout to zero disables the verification of synthesized terms."
```

Reply:
Right, I agree, it disables the timeout, the message doesnt need to change.

Reply:
Ah, sorry. I was confused about the semantics you wanted to have in the end. You are right...